controlplane/kubeadm/internal/etcd/etcd.go

```diff
@@ -156,16 +156,16 @@ func (c *Client) Members(ctx context.Context) ([]*Member, error) {
 	}
 
 	clusterID := response.Header.GetClusterId()
-	members := make([]*Member, len(response.Members))
-	for i, m := range response.Members {
+	members := make([]*Member, 0)
```

Comment:
```suggestion
	members := make([]*Member, 0, len(response.Members))
```

If you wanted to pre-allocate slots, but not grow the array and use `append`.